front/app/components/ContentUploadDisclaimer/index.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+
+// components
+import {
+  IconTooltip,
+  Text,
+  useBreakpoint,
+} from '@citizenlab/cl2-component-library';
+import Modal from 'components/UI/Modal';
+
+// intl
+import { useIntl } from 'utils/cl-intl';
+import messages from './messages';
+
+const ContentUploadDisclaimerTooltip = () => {
+  const { formatMessage } = useIntl();
+  const isMobileOrSmaller = useBreakpoint('phone');
+  const [showFullDisclaimer, setShowFullDisclaimer] = useState(false);
+
+  return (
+    <>
+      <IconTooltip
+        content={
+          <>
+            <Text m="0px" color="white" fontSize="s">
+              {formatMessage(messages.contentUploadDisclaimerTooltip)}
+            </Text>
+            <Text
+              onClick={(e) => {
+                e.preventDefault();
+                setShowFullDisclaimer(true);
+              }}
+              color="white"
+              fontSize="s"
+              m="0px"
+              mt="8px"
+              role="button"
+              style={{ cursor: 'pointer' }}
+              tabIndex={0}
+            >
+              <span style={{ textDecoration: 'underline' }}>
+                {formatMessage(messages.readFullDisclaimer)}
+              </span>
+              .
```

Comment:
I didn't mean to be such a PITA about a dot, but it should rather be part of the message. If you translate this to Arabic (right to left) for example, it'll be hard to get this right.

Since it's already been added to Crowdin, I think it's OK to leave as is and fix when we get complaints (actual Arabic-speaking people noticing it). But good to be aware of. :-)

Reply:
Ahh ok! I think it might be hard to remove it from the underlined text if it's part of the same message though :thinking: Or do you know of a way to do that? :slightly_smiling_face: